lib/include/srsran/interfaces/sched_interface.h

```diff
@@ -40,7 +40,7 @@ class sched_interface
 
   const static int MAX_SIB_PAYLOAD_LEN = 2048;
   const static int MAX_SIBS            = 16;
-  const static int MAX_LC              = 11;
+  const static int MAX_LC              = 14;
```

Comment:
can you revert this change after you rebase

Reply:
Done